### ISSUE TITLE
You can update the mpv version now that ffmpeg 4.0 is out

### ISSUE THREAD

**Issue by yetisyny** [lhc70000/iina#1646]
ffmpeg 4.0 is out so you can update IINA to use the latest mpv version (currently 0.28.2). No need to stay on an older ffmpeg version like 0.27 anymore like you’ve been doing, I know for awhile the latest mpv version required an ffmpeg version that wasn’t out yet but all of that is now fixed upstream with ffmpeg 4.0 being out. Obviously you would need to compile against ffmpeg 4.0 too to use the latest mpv in IINA. It’s nice that all the upstream issues with mpv and ffmpeg are now resolved, makes things easier for you, now you can use the latest versions of those dependencies without any problems.

Anyway mpv 0.28.2 has a whole long list of improvements and bug fixes over mpv 0.27 so this should translate into a bunch of improvements and bug fixes in IINA now that you can finally use it with a stable version of ffmpeg. Also ffmpeg 4.0 has a bunch of neat improvements too so this should be all-around pretty awesome and get IINA to work better too.

**Comment by lhc70000** [comment]
I'm [working on this](https://github.com/lhc70000/iina/tree/mpv-0.28.2). It may still need some time since mpv 0.28 changed its render API.

**Comment by yetisyny** [comment]
Good to know you are working on updating IINA to work with those versions! IINA is the best video player for Macs right now and I don’t say that lightly, only one program can be best. Oh and I waited until both the latest mpv and ffmpeg had been updated and were working together on Homebrew before posting this. I didn’t want to post something like this before that had been tested on Macs by the Homebrew team.
